crates/aptos/src/account/multisig_account.rs

```diff
@@ -40,6 +40,8 @@ pub struct Create {
     pub(crate) additional_owners: Vec<AccountAddress>,
     /// The number of signatures (approvals or rejections) required to execute or remove a proposed
     /// transaction.
+	#[clap(long)]
+	pub(crate) timeout_duration: u64,
```

Comment:
this line should go before doc comment for `num_signatures_required`

Reply:
do we require change in cli for testing the supra_framework?

Reply:
CLI part has been discussed. Declaration order and description added to resolve concern.

Reply:
there is a similar Issue in smr-moonshot https://github.com/Entropy-Foundation/smr-moonshot/issues/538

i will add the summary of the discussion to that issue comment